backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -193,8 +192,15 @@ private ApiUser getApiUser(UUID id, Boolean includeArchived) {
     return result;
   }
 
+  // current user is checked using their token
+  private boolean isCurrentUserAdmin() {
```

Comment:
`isCurrentUserSiteAdmin` or -`GlobalAdmin` please, to distinguish from org admin